WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -1276,6 +1276,13 @@ private extension AztecPostViewController {
         }
 
         alert.addCancelActionWithTitle(MoreSheetAlert.keepEditingTitle)
+
+        if FeatureFlag.asyncPosting.enabled {
+            alert.addDefaultActionWithTitle("Async Upload") { [unowned self]  _ in
```

Comment:
Let's make it clear that this is in development (I know it's for local developer only, but just let's make it super obvious) – call it `Async Upload (Debug)`?

Reply:
I wonder as well if we should move this to the top of the list – so it's less 'in the way' for people wanting to use the regular options there.